### PR TITLE
Additions and modifications to inventory of past, current and upcoming versions

### PR DESCRIPTION
**Pull request by antrim**
Moving towards full implementation of #188 - GBFS documentation versioning and and feed conformance.

Remaining to do:
- [ ] Establish that feed producers and consumers are supporting v1.1 #25 and #181 
- [ ] Tag v1.1 after this PR is merged
- [ ] Create a MAJOR Release Candidate branch that marks beta/experimental fields

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,9 +26,23 @@ The specification has been designed with the following concepts in mind:
...
-
-* [Current spec](gbfs.md)
-* [Original draft spec in a Google doc](https://docs.google.com/document/d/1BQPZCKpem4-n6lUQDD4Mi8E5hNZ0-lhY62IVtWuyhec/edit#heading=h.ic7i1m4gcev7) (reference only)
+* [v1.0](https://github.com/NABSA/gbfs/blob/df473ca4adbff982d67b50ac00b625191591d8f8/gbfs.md)
+* [v1.1: MINOR Release Candidate](https://github.com/NABSA/gbfs/blob/master/gbfs.md)
+  * #25 - Add deep links for iOS, Android, and web apps
+  * #181 - Add feed\_contact\_email field to system\_information.json
+  * #188 - GBFS documentation versioning and and feed conformance
+* [v2.0: MAJOR Release Candidate](https://github.com/NABSA/gbfs/labels/v2.0)
+  * #182 - Require license\_url, add attribution fields PR #182
```
Comment (heidiguenin) [inline README.md]: Don't need #182 in here twice.

File: gbfs.md [hunk 2]
```diff
@@ -512,17 +509,6 @@ Note that the Android URI and iOS Universal Link URLs don’t necessarily use th
    ...
 ```
 
-## Possible Future Enhancements
```
Comment (heidiguenin) [inline gbfs.md:515]: Also delete line 25 that refers to this section.

File: README.md [hunk 3]
```diff
@@ -25,10 +25,22 @@ The specification has been designed with the following concepts in mind:
...
+  * 2015 November 05 - GBFS V1.0 Adopted by NABSA board - [Original draft spec in a Google doc](https://docs.google.com/document/d/1BQPZCKpem4-n6lUQDD4Mi8E5hNZ0-lhY62IVtWuyhec/edit#heading=h.ic7i1m4gcev7) (reference only)
+  * 2015 August - Latest changes incorporated and name change to GBFS (comments from Motivate, 8D, others)
+  * 2015 June - Proposed refinements (prepared by Jesse Chan-Norris on behalf of Motivate)
+  * 2015 January - NABSA Draft (prepared by Mitch Vars)
+* **[v1.1: MINOR Release Candidate](https://github.com/NABSA/gbfs/blob/master/gbfs.md)**
+  * [#25](https://github.com/NABSA/gbfs/pull/25) - Add deep links for iOS, Android, and web apps
+  * [#181](https://github.com/NABSA/gbfs/pull/181) - Add feed\_contact\_email field to system\_information.json
+  * [#188](https://github.com/NABSA/gbfs/pull/188) - GBFS documentation versioning and and feed conformance
+* **[v2.0: MAJOR Release Candidate](https://github.com/NABSA/gbfs/labels/v2.0)**
+  * [#182](https://github.com/NABSA/gbfs/pull/182) - Require license\_url, add attribution fields PR #182
```
Comment (heidiguenin) [inline README.md]: Delete second reference to PR #182

**Review by heidiguenin — CHANGES_REQUESTED** [review]
Small edits

File: README.md (hunk 3 above)
Comment (antrim) [reply README.md]: Fixed

**Review by antrim — COMMENTED** [review]
Made corrections requested by @heidiguenin

**Pull request merged**